app/src/main/java/com/github/orkest/View/auth/SignUpForm.kt

```diff
@@ -65,9 +88,8 @@ fun SignUpForm(navController: NavController, viewModel: AuthViewModel) {
 
                 Spacer(modifier = Modifier.height(15.dp))
 
+                // Box with button to choose the service provider from a dropDownMenu
                 Box {
-
-
                     Button(
```

Comment:
Same comment as before, I'd store this block of code in a function that you'd then call here.
I'd say it makes it easier to read and maintain the code

Reply:
Apparently I am incapable of selecting lines correctly, let me try again...